rachelbuchta/refactor_tractor
# Feat/display images

**What does this PR do?**

* imported images into scripts file so that they appear on the site
* Added a helper method to create html elements for each tag in every recipe and display it on the recipes card so that the filter by tag check boxes rendered the correct recipes
* Added new styling to the css for recipe cards to be set up with a grid in order to display the tags in a nice way.

**What does it fix?**

* Fixes the bug that didn't filter recipes correctly since not all of the tags for each recipe were showing up

**Is this a feature or a fix?**

* Fix

**Where should the reviewer start?**

* Line 75 in scripts.js

**How should this be tested?**

* Open up the browser and attempt checking off boxes for whatever tag of recipes you want to see

**Do you have any questions or issues about this addition?**

* Will definitely still need to do some. refactoring but it works!
